src/client/components/App/index.js

```diff
@@ -60,7 +57,7 @@ const App = () => {
         />
       )}
       {currentTab === constants.TABS.COURSE_RESERVES && (
-        <CourseReserves url={courseData.url} />
+        <CourseReserves url="" />
```

Comment:
@kilometerskang Can you create a ticket to make this url pull from the database?